### ISSUE TITLE
通知区域（托盘）菜单项增加访问键

### ISSUE THREAD

**Issue by Geeyun-JY3** [Snipaste/feedback#2048]
即类似贴图的助记键，就是显示的位置在语言为中文时得在选项文字之后，而不是像贴图的右键菜单一样。

**Comment by Geeyun-JY3** [comment]
话说为什么会专门做出贴图的助记键这样一个设置项来，是不喜欢中文环境下传统的在设置项文字后用括号括住字母这种助记键显示方式吗

**Comment by AnLazyOtter** [comment]
如图，你是说这种吗？
![Snipaste_2021-12-05_22-37-26](https://user-images.githubusercontent.com/40395191/144751116-4aea5304-604b-4785-8ebe-9e84d7ea0a4d.png)

**Comment by Geeyun-JY3** [comment]
当然不是。
右边的只是快捷键提示，对应的快捷键一般是应用内全局生效的。
而助记（访问）键（mnemonic / access keys）是用于导航至可见用户界面元素的，只在当前用户界面元素生效。以本 issue 的预期效果为例，就是有助记（访问）键导航至通知区域（托盘）菜单的各个菜单项，只在通知区域（托盘）菜单中生效。

传统 Windows 应用的助记键一般会取自菜单项英文文字中的某个字符。
在界面语言为英文时，传统 Windows 应用的助记键对应字符会有下划线（在 Windows 10 设置应用的「轻松使用 > 键盘」中有一个「在访问键(如果可用)下添加下划线」设置项可以控制下划线是否显示）：
![image](https://user-images.githubusercontent.com/49396421/144753194-99aa921c-23d1-41e9-8c5f-359451d6d33b.png)

在界面语言为简中时，传统 Windows 应用的助记键会显示在菜单项文字最后并用半角括号括住（所以上文提到的「在访问键(如果可用)下添加下划线」设置项在应用界面语言为简中且本地化完整时是没什么用的）：
![image](https://user-images.githubusercontent.com/49396421/144753480-7f083e53-01eb-4bea-8d85-05244533008b.png)

不严谨地说，用户在某个菜单中按下助记（访问）键（一般对应单个字母/数字）即相当于点击对应菜单项。

另外，Snipaste 贴图的右键菜单的助记（访问）键有些特殊，和传统 Windows 应用的显示方式不太一样，反而类似于你截图中的快捷键提示的显示方式（在选项文字右方），也有一个专门的「在右键菜单显示助记键」设置项。我[上面](https://github.com/Snipaste/feedback/issues/2048#issuecomment-985122315)也有提到不明白开发者这么做的用意。

通知区域（托盘）菜单示例（以下菜单项中的省略号的位置有些小问题，但不影响使用）：
[![WinDynamicDesktop](https://user-images.githubusercontent.com/49396421/144771289-93055eee-7b5e-47ec-8b6b-d1cb27b13370.png)](https://github.com/t1m0thyj/WinDynamicDesktop "WinDynamicDesktop") 

参考：
- https://docs.microsoft.com/en-us/windows/apps/design/input/access-keys
- https://docs.microsoft.com/en-us/windows/apps/design/input/keyboard-accelerators
- https://docs.microsoft.com/en-us/windows/apps/design/accessibility/keyboard-accessibility#keyboard-shortcuts

**Comment by liulex** [comment]
@Geeyun-JY3 确实 Snipaste 贴图助记键的显示跟传统方式不一样，没有特别的原因，就是我个人觉得右对齐更好看。而之所以特意加个选项，是因为大部分人是不需要的，不显示出来更简洁。

托盘菜单处如果加助记键，就需要跟全局快捷键区分开来，比如加个括号：
![](https://user-images.githubusercontent.com/2010459/145239597-23fc651f-ab48-4194-a2ce-3914baaaf9f3.png)

但我觉得这样也挺难看的。。而且如果这里的助记键加了括号，那贴图的右键菜单也要加上了。

**Comment by Geeyun-JY3** [comment]
我感觉如果都右对齐了就容易让用户觉得这里的助记键和上面的应用内全局快捷键是一类或者类似的东西了（虽说现在用户如果把托盘菜单的快捷键提示和贴图右键菜单的助记键提示联系起来也容易得出类似的结论），逻辑上就不对劲。而且我也觉得这样加了括号再右对齐还是挺难看的。所以总的来说我个人是不建议这样做的。

如果不使用传统的助记键显示方式为必要前提，那就把这个 issue 关了吧。
